commitrmnocb/report.go

```diff
@@ -0,0 +1,84 @@
+package commitrmnocb
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+
+	cciptypes "github.com/smartcontractkit/chainlink-common/pkg/types/ccipocr3"
+
+	"github.com/smartcontractkit/chainlink-ccip/commit"
+)
+
+func (p *Plugin) Reports(seqNr uint64, outcomeBytes ocr3types.Outcome) ([]ocr3types.ReportWithInfo[[]byte], error) {
+	outcome, err := DecodeCommitPluginOutcome(outcomeBytes)
+	if err != nil {
+		// TODO: metrics
+		p.lggr.Errorw("failed to decode Outcome", "outcomeBytes", outcomeBytes, "err", err)
+		return nil, fmt.Errorf("failed to decode Outcome: %w", err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("failed to decode Outcome (%s): %w", hex.EncodeToString(outcomeBytes), err)
```

Including the outcome data for debugging